added docker based env (#863)
Graylog2/documentation: README.md

	# pip install -r requirements.txt
	
Once the above is done you are prepared to contribute to the documentation and preview the work live in your local browser. See the daily usage chapter.

#### Docker

> To easy up the contribution to the documentation you can also use Docker to create a local environment for the documentation. The image is build locally and use the current base dir of this repository for creating and serving the documentation. This way their is not need to handle virtual environments. Just build the image and run. 

Reviewer: ```suggestion
To ease up contributions to the documentation, you can also use Docker to create a local environment for the documentation. The image is built locally and uses the current base dir of this repository for creating and serving the documentation. This way, there's no need to handle virtual environments. Just build the image and run. 
```

    docker build -t graylog/documentation -f Dockerfile  .
    docker run -it -d --rm -v .:/web -u $(id -u):$(id -g) -p 8000:8000 --name graylog/documentation graylog-documentation

##### docker_run.sh (helper)

> If unsure how to run what command with docker, the simple script `docker_run.sh` was created. This runs on Mac and Linux (maybe Windows when bash is available). The simplification is done in a way that you just need ro run the command. If necessary it will build the image, and start the docker container. If the container is up and running it will ask if the container should be stopped. In addition it will be ask if the image should be removed too. 

Reviewer: ```suggestion
If unsure how to run what command with docker, the simple script `docker_run.sh` was created. This runs on Mac and Linux (maybe Windows when bash is available). The simplification is done in a way that you just need to run the command. If necessary it will build the image, and start the docker container. If the container is up and running it will ask if the container should be stopped. In addition it will be ask if the image should be removed too. 
```
